.github/workflows/validate.yml

```diff
@@ -0,0 +1,19 @@
+# Validates the AsyncAPI definition
+
+name: asyncapi_validate
```

Comment:
Could we change the case to match all the other CI jobs?

![image](https://github.com/DiamondLightSource/blueapi/assets/29771545/1ca7829e-ea63-4476-ba37-b3afa02e8e68)

```suggestion
name: AsyncAPI Validate
```